sdk/src/test/scala/io/horizen/account/api/rpc/service/EthServiceTest.scala

```diff
@@ -1292,4 +1292,33 @@ class EthServiceTest extends JUnitSuite with MockitoSugar with ReceiptFixture wi
   def txpool_inspect(): Unit = {
     assertJsonEquals(txPoolInspectOutput, rpc("txpool_inspect"))
   }
+
+  @Test
+  def eth_getSHA3(): Unit = {
+    val validCases = Table(
+      ("Input", "Expected output"),
+      ("0x68656c6c6f20776f726c64", "\"0x47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad\""),
+      ("0x68656C6C6F20776F726C64", "\"0x47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad\""),
+      ("0x", "\"0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470\""),
+    )
+
+    val invalidCases = Table("Input",
```

Comment:
It would be nice to have some comments, explaining why some cases are not valid, at least for non obvious ones.

Reply:
Done